curriculum/challenges/english/25-front-end-development/lecture-working-with-regular-expressions/6733c5ba834ded4bb067e67c.md

```diff
@@ -1,14 +1,156 @@
 ---
 id: 6733c5ba834ded4bb067e67c
 title: What Are Some Common Regular Expression Modifiers Used for Searching?
```

Comment:
finished editing for this one. need to check the video before uploading to youtube. We can merge this in for now. But there will need to be another PR to add in the id.